### PR TITLE
established placeholder for driver page

### PR DESCRIPTION
**Pull request by pabucsb**
In this PR I made additions to our frontend and added a placeholder for driver dashboard. The dropdown menu is only visible for those with the driver role. 

image

image

### CODE REVIEW

File: frontend/src/main/components/Ride/RideForm.js [hunk 1]
```diff
@@ -147,21 +147,22 @@ function RideForm({ initialContents, submitAction, buttonLabel = "Create" }) {
...
-                    {...register("room", {
-                        required: "Room number is required."
-                    })}
-                    placeholder="e.g. 2225"  
-                    defaultValue={initialContents?.room} 
+                    {...register("dropoffRoom")}
+                    isInvalid={Boolean(errors.dropoffRoom)}
+                    //{...register("dropoffRoom", {
+                        //required: "Drop OffRoom number is required."
+                    //})}
```
Comment (Aasish-Virjala) [inline frontend/src/main/components/Ride/RideForm.js:159]: Remove comments
```suggestion
```

File: frontend/src/main/components/Ride/RideForm.js [hunk 2]
```diff
@@ -182,6 +183,43 @@ function RideForm({ initialContents, submitAction, buttonLabel = "Create" }) {
...
+                <Form.Label htmlFor="pickup-room">Pickup Room</Form.Label>
+                <Form.Control
+                    data-testid={testIdPrefix + "-pickup-room"}
+                    id="pickup-room"
+                    type="text"
+                    {...register("pickupRoom")}
+                    isInvalid={Boolean(errors.pickupRoom)}
+                    //{...register("dropoffRoom", {
+                        //required: "Drop OffRoom number is required."
+                    //})}
```
Comment (Aasish-Virjala) [inline frontend/src/main/components/Ride/RideForm.js:196]: Remove comments
```suggestion
```

File: frontend/src/main/components/Ride/RideForm.js [hunk 3]
```diff
@@ -182,6 +183,43 @@ function RideForm({ initialContents, submitAction, buttonLabel = "Create" }) {
...
+                <Form.Label htmlFor="notes">Notes</Form.Label>
+                <Form.Control
+                    data-testid={testIdPrefix + "-notes"}
+                    id="notes"
+                    type="text"
+                    {...register("notes")}
+                    isInvalid={Boolean(errors.notes)}
+                    //{...register("dropoffRoom", {
+                        //required: "Drop OffRoom number is required."
+                    //})}
```
Comment (Aasish-Virjala) [inline frontend/src/main/components/Ride/RideForm.js:215]:
```suggestion
```

File: src/main/java/edu/ucsb/cs156/gauchoride/controllers/RideController.java [hunk 4]
```diff
@@ -90,25 +90,35 @@ public Ride postRide(
...
+        @Parameter(name="notes", description="String, Notes for additional information", example="Library entrance by the arbor", required = false)
+        @RequestParam String notes
+
+
         )
         {
 
         Ride ride = new Ride();
-        
+        //not sure if we have to set dropoffroom and pickuproom
```
Comment (Aasish-Virjala) [inline src/main/java/edu/ucsb/cs156/gauchoride/controllers/RideController.java:110]:
```suggestion
```

**Review by Aasish-Virjala — CHANGES_REQUESTED** [review]
Just remove comments and I think it's good.

**Review by Aasish-Virjala — DISMISSED** [review]
LGTM

**Review by adhit-s — APPROVED** [review]
Everything looks good. My PR for the rider dashboard will remove the RideForm.js file, but it doesn't seem like this issue is using that so we're in the clear. Approved!

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**